### ISSUE TITLE
Test on iOS physical devices in LUCI

### ISSUE THREAD

**Issue by cyanglaz** [flutter/flutter#68526]
LUCI is currently only running on iOS simulator because we haven't been able to figure out how to set up iOS signing on LUCI.

**Comment by godofredoc** [comment]
Is this a request? we don't have physical devices on luci.

**Comment by cyanglaz** [comment]
@godofredoc Yeah it was a request. I didn't know much about LUCI but if it is impossible, we can close it.

**Comment by godofredoc** [comment]
We are moving the devicelab devices to LUCI this quarter those test beds will support signing.

**Comment by cyanglaz** [comment]
@godofredoc Cool, once that happens, the tests running on LUCI can run on real devices.

**Comment by godofredoc** [comment]
@cyanglaz devicelab has been fully migrated to luci. Do we need to track something else in this bug or can we close it?
